src/components/menu/menu.tsx

```diff
@@ -262,8 +274,7 @@ export class Menu {
         }
     }
 
-    @Watch('currentSubMenu')
-    protected currentSubMenuWatcher() {
+    private getBreadcrumbsItems() {
```

Comment:
Will this still reflect the content of the menu now that we're not longer watching for changes and re-rendering the breadcrumbs?

Reply:
Yes, because currentSubMenu is a (mutable) prop. So we will always re-render when it changes, and we will always call `this.getBreadcrumbsItems()` when rendering. It simplifies things to remove the watcher, and it also means breadcrumbs are rendered on first render, rather than only when `currentSubMenu` changes.